com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -508,6 +508,19 @@ public struct ConnectionApprovalRequest
 
         internal static event Action OnSingletonReady;
 
+#if UNITY_EDITOR
+        internal delegate void ResetNetworkManagerDelegate(NetworkManager manager);
+
+        internal static ResetNetworkManagerDelegate OnNetworkManagerReset;
+#endif
+
+        private void Reset()
```

Comment:
**Question:** Was this meant to be invoked from somewhere? I don't see any calls to it.

Reply:
This is a Unity event like Awake, Update, etc. It's called when the component is created, as well as when the user hits "reset" on the component in the inspector.

Reply:
Oh, I totally forgot about that! Thanks :)